sidebars.js

```diff
@@ -1,59 +1,40 @@
 module.exports = {
   sidebar: [
-    {
-      type: 'doc',
-      id: 'about-unum-id',
-    },
-    {
-      type: 'category',
-      label: 'Introduction',
-      /* should collapse when we have lots of docs */
-      collapsed: false,
-      items: [
-        'terminology',
-        'faq-info',
-      ],
-    },
     {
       type: 'category',
-      label: 'Guides',
-      /* should collapse when we have lots of docs */
+      label: 'Solutions',
       collapsed: false,
       items: [
-        'acceptance-guide',
-        'issuance-guide'
+        '1-click-signup',
+        'issue-to-earn',
       ],
     },
     {
       type: 'category',
-      label: 'Components',
-      /* should collapse when we have lots of docs */
+      label: 'Resources',
       collapsed: false,
       items: [
-        'api-overview',
+        'terminology',
+        {
+          type: 'category',
+          label: 'API',
+          collapsed: false,
+          items: [
+            {
+              type: 'doc',
+              id: 'api-overview',
+            },
+            {
+              type: 'link',
+              label: 'Postman Collection',
+              href: 'https://api.docs.verified.inc/'
+            },
+          ],
+        },
         'schema',
+        'demos',
```

Comment:
is demos left on here on purpose? It is empty. Why not mention Kredita?

Reply:
Ah whoops. Yeah will plan to mention Kredita. Need to get to that